cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt

```diff
@@ -142,24 +142,26 @@ class ClearlyDefinedUploadCommand : CliktCommand(
             )
         }
 
-        var error = false
+        var uploadedCurationsCount = 0
         val uploadableCurations = curationsByHarvestStatus[HarvestStatus.HARVESTED].orEmpty() +
                 curationsByHarvestStatus[HarvestStatus.PARTIALLY_HARVESTED].orEmpty()
 
         uploadableCurations.forEachIndexed { index, curation ->
             print("Curation ${index + 1} of ${uploadableCurations.size} for package '${curation.id.toCoordinates()}' ")
 
             when (val summary = putCuration(curation)) {
-                null -> {
-                    println("failed to be uploaded.")
-                    error = true
-                }
+                null -> println("failed to be uploaded.")
+                else -> {
+                    println("was uploaded successfully:\n${summary.url}")
 
-                else -> println("was uploaded successfully:\n${summary.url}")
+                    uploadedCurationsCount++
+                }
             }
         }
 
-        if (error) {
+        println("Uploaded $uploadedCurationsCount / ${uploadableCurations.size} curations.")
```

Comment:
Nit: Mind spelling out "/" as "of" like we do in the `ReporterCommand`?

Reply:
Done. I've also added 'Successfully' like in the `ReporterCommand`.